src/ducks/notifications/LateHealthReimbursement.js

```diff
@@ -0,0 +1,210 @@
+import Handlebars from 'handlebars'
+import Notification from './Notification'
+import { isHealthExpense } from '../categories/helpers'
+import htmlTemplate from './html/late-health-reimbursement-html'
+import * as utils from './html/utils'
+import { keyBy, uniq } from 'lodash'
+import logger from 'cozy-logger'
+import { BankTransaction, BankAccount } from 'cozy-doctypes'
+import { isReimbursementLate } from 'ducks/transactions/helpers'
+import { subDays, subMonths, format as formatDate } from 'date-fns'
+import { Bill } from 'models'
+import { getReimbursementBillId, getReimbursementBillIds } from './helpers'
+
+const log = logger.namespace('lateHealthReimbursement')
+
+const ACCOUNT_SEL = '.js-account'
+const DATE_SEL = '.js-date'
+const TRANSACTION_SEL = '.js-transaction'
+
+const toText = cozyHTMLEmail => {
+  const getTextTransactionRow = $row =>
+    $row
+      .find('td')
+      .map((i, td) =>
+        $row
+          .find(td)
+          .text()
+          .trim()
+      )
+      .toArray()
+      .join(' ')
+      .replace(/\n/g, '')
+      .replace(' €', '€')
+      .trim()
+
+  const getContent = $ =>
+    $([ACCOUNT_SEL, DATE_SEL, TRANSACTION_SEL].join(', '))
+      .toArray()
+      .map(node => {
+        const $node = $(node)
+        if ($node.is(ACCOUNT_SEL)) {
+          return '\n\n### ' + $node.text()
+        } else if ($node.is(DATE_SEL)) {
+          return '\n' + $node.text() + '\n'
+        } else if ($node.is(TRANSACTION_SEL)) {
+          return '- ' + getTextTransactionRow($node)
+        }
+      })
+      .join('\n')
+  return utils.toText(cozyHTMLEmail, getContent)
+}
+
+class LateHealthReimbursement extends Notification {
+  constructor(config) {
+    super(config)
+    this.interval = config.value
+  }
+
+  isAlreadyNotified(transaction) {
+    return (
+      transaction.cozyMetadata &&
+      transaction.cozyMetadata.notifications &&
+      transaction.cozyMetadata.notifications[LateHealthReimbursement.settingKey]
+    )
+  }
+
+  async getTransactions() {
+    const DATE_FORMAT = 'YYYY-MM-DD'
+    const today = new Date()
+    const lt = formatDate(subDays(today, this.interval), DATE_FORMAT)
+    const gt = formatDate(subMonths(lt, 6), DATE_FORMAT)
+
+    log('info', `Fetching transactions between ${gt} and ${lt}`)
+    const transactionsInDateRange = await BankTransaction.queryAll({
+      date: {
+        $gt: gt,
+        $lt: lt
+      }
+    })
+    log(
+      'info',
+      `Fetched ${
```

Comment:
Number at the beginning of the log to keep the same convention as we discussed ;)